rosidl_parser/rosidl_parser/__init__.py

```diff
@@ -445,7 +445,7 @@ def parse_value_string(type_, value_string):
     if type_.is_primitive_type() and not type_.is_array:
         return parse_primitive_value_string(type_, value_string)
 
-    if type_.is_primitive_type() and type_.is_array and type_.type != 'string':
+    if type_.is_primitive_type() and type_.is_array:
```

Comment:
The following code doesn't work for strings yet since it doesn't consider quoting / escaping so this can not be enabled yet.